demisto_sdk/commands/validate/default_config.toml

```diff
@@ -1,21 +1,21 @@
 [use_git]
 
 select = [
-    "BA101", "BA105", "BA106", "BA116", "BA118",
-    "PA107", "PA108", "PA111", "PA115", "PA130",
-    "BC100",
-    "GR106",
-    "DO108",
-    "IN108",
-    "IN130",
-    "PB118",
+#    "BA101", "BA105", "BA106", "BA116", "BA118",
```

Comment:
Why?

Reply:
just for testing, thanks for reminding me to remove it, ill revert

Reply:
reverted :)